src/otx/algo/detection/losses/yolov9_loss.py

```diff
@@ -405,6 +405,10 @@ def forward(
         Returns:
             dict[str, Tensor]: The loss dictionary.
         """
+        if targets.shape[1] == 0:
```

Comment:
Could you explain what it's for?

Reply:
`targets` has (batch, num_bboxes, 5).

Reply:
Won't it destroy the training? Can't we just raise an error?

Reply:
I added below to skip the iteration
https://github.com/openvinotoolkit/training_extensions/blob/2fb694e3d9907e2dd3ecb9442ba2184166361758/src/otx/core/model/base.py#L146-L150

Reply:
Thanks for explanation. Could you add unit test for this case?

Reply:
Is it expected that model returns None sometimes?

Reply:
> Is it expected that model returns None sometimes?

Just for yolov9. If other models could return None, TypeError would be raised.
https://github.com/openvinotoolkit/training_extensions/blob/2fb694e3d9907e2dd3ecb9442ba2184166361758/src/otx/core/model/base.py#L181

Reply:
> Thanks for explanation. Could you add unit test for this case?

Done at dce91cd